ratelimit/ratelimit.go

```diff
@@ -338,8 +338,14 @@ func (s Settings) String() string {
 type limiter interface {
 	// Allow is used to get a decision if you should allow the
 	// call to pass or to ratelimit
+	//
+	// Depreceated
```

Comment:
Depreceated -> Deprecated

Reply:
I didn't delete it but if you like I can, I left it in favour of deprecation first then deletion.

Reply:
yes please delete

Reply:
From implementations also or just the interface?